codes/swift/chapter_stack_and_queue/deque.swift

```diff
@@ -0,0 +1,43 @@
+/**
+ * File: deque.swift
+ * Created Time: 2023-01-14
+ * Author: nuomi1 (nuomi1@qq.com)
+ */
+
+@main
+enum Deque {
+    /* Driver Code */
+    static func main() {
+        /* 初始化双向队列 */
+        // Swift 没有内置的双向队列类，可以把 Array 当作双向队列来使用
```

Comment:
Is the complexity of `poolFirst()` O(n)? If so, please add a comment to mention it.

Reply:
Same for `queue.swift`